CMakeLists.txt

```diff
@@ -45,6 +45,15 @@ endif()
 set(MPI_CXX_SKIP_MPICXX ON)
 find_package(MPI REQUIRED)
 
+
+find_package(OpenMP REQUIRED)
+foreach(lang IN ITEMS CXX C)
+  get_property(tmp TARGET OpenMP::OpenMP_${lang} PROPERTY INTERFACE_COMPILE_OPTIONS)
+  list(TRANSFORM tmp REPLACE "-fexceptions" "$<$<COMPILE_LANGUAGE:CUDA>:SHELL:-Xcompiler >-fexceptions")
+  set_property(TARGET OpenMP::OpenMP_${lang} PROPERTY INTERFACE_COMPILE_OPTIONS "${tmp}")
+  unset(tmp)
+endforeach()
+
```

Comment:
I'd rather not have OpenMP in there unless we really want to use it. Do you think you'll need it any time soon? Generally, the more dependencies, the more ways for things to go wrong...

Reply:
I put it in because there is one section of libpsc wrapped in parallel for that we can use.

Reply:
So I think this part shouldn't be there standalone then, but in a PR that actually adds something that uses OpenMP.

Also, I'd guess that you copied the subsequent `foreach` stuff from MPI just because it was there, or did you in fact get errors due `-fexceptions` from nvcc?

Reply:
Oh no, I straight copied it.  maybe this would be better 
`find_package(OpenMP)
if(OpenMP_CXX_FOUND)
    target_link_libraries(MyTarget PUBLIC OpenMP::OpenMP_CXX)
endif()`
But either way we can just leave it out until there is openmp

Reply:
Yes, that's the right way (with the link part moved to `src/libpsc`)